src/sql/workbench/contrib/notebook/test/electron-browser/notebookFindModel.test.ts

```diff
@@ -280,7 +280,7 @@ suite('Notebook Find Model', function (): void {
 		mockContentManager.setup(c => c.loadContent()).returns(() => Promise.resolve(contents));
 		defaultModelOptions.contentManager = mockContentManager.object;
 		// Initialize the model
-		model = new NotebookModel(defaultModelOptions, undefined, logService, undefined, undefined);
+		model = new NotebookModel(defaultModelOptions, undefined, logService, undefined, undefined, undefined);
```

Comment:
>undefined [](start = 94, length = 9)

A bunch of tests are failing now because this is undefined. Suggest using `new TestContextService()` for this instead.